packaging/macos/sign_notarize_staple.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+[ -z "${APPLE_ID_USERNAME}" ] && echo "Please set the $APPLE_ID_USERNAME env var." >&2 && exit 1
+[ -z "${APPLE_ID_PASSWORD}" ] && echo "Please set the $APPLE_ID_PASSWORD env var." >&2 && exit 1
+[ -z "${ASC_PROVIDER}" ] && echo "Please set the $ASC_PROVIDER env var." >&2 && exit 1
+[ -z "${DMG_FILE}" ] && echo "Please set the $DMG_FILE env var." >&2 && exit 1
+
+# sign DMG
+codesign --verbose=4 --options runtime \
+    --sign "${APPLE_CODESIGN_IDENTITY}" "$(dirname $0)/entitlements.plist" $DMG_FILE
```

Comment:
```suggestion
    --sign "${APPLE_CODESIGN_IDENTITY}" "$(dirname $0)/entitlements.plist" "${DMG_FILE}"
```

to prevent word splitting if there are spaces in the path.